### PR TITLE
BDRSPS-556 Created survey occurrence instructions markdown template

### PR DESCRIPTION
**Pull request by joecrowleygaia**
and associated rendering logic. This also fixes the issue caused by removing KINGDOM controlled vocabulary from the occurrence mappers, by altering the row generation logic for the controlled vocabs' tables.

### CODE REVIEW

File: abis_mapping/templates/survey_occurrence_data/templates/instructions.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# SYSTEMATIC SURVEY OCCURRENCES DATA TEMPLATE INSTRUCTIONS
+
+## OVERVIEW
+Use this template to record occurrence data; that is the presence or absence of an organism
+at a particular site locality at a point in time.
+
+This Systematic Survey Occurrences template **must be used in combination** with the
+Systematic Survey Metadata template and, in some cases, the Systematice Survey Sites
```
Comment (serge-gaia) [inline abis_mapping/templates/survey_occurrence_data/templates/instructions.md:8]: typo: Systematice

File: abis_mapping/templates/survey_occurrence_data/templates/instructions.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+E.g. eventRemarks, associatedTaxa, pathway.
+
+<ins>Table 1: Systematic Survey Occurrence data template fields with descriptions, conditions,
+datatype format, and examples.</ins>
+
+{{tables.fields}}
+
+## APPENDICES
+### APPENDIX-I: VOCABULARY LIST
+Apart from geodeticDatum, the data validation doesnot require adherence tho the below vocabularies
```
Comment (serge-gaia) [inline abis_mapping/templates/survey_occurrence_data/templates/instructions.md:75]: typo: doesnot

File: abis_mapping/templates/survey_occurrence_data/templates/instructions.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+for each of the fields indicated as having vocabularies. These vocuabularies are provided as a
```
Comment (serge-gaia) [inline abis_mapping/templates/survey_occurrence_data/templates/instructions.md:76]: typo: vocuabularies

**Comment by serge-gaia** [comment]
a few typos that also apply to the the incidental occurrences

**Comment by serge-gaia** [comment]
I have also noticed that the schema example for the field `threatStatusCheckProtocol` there's a link that is not properly formatted. It contains spaces: https://github.com/gaiaresources/abis-mapping/blob/bf36190055339a9fed76f5fc31a7d08189d1f2b6/abis_mapping/templates/survey_occurrence_data/schema.json#L611

File: abis_mapping/templates/survey_occurrence_data/templates/instructions.md (hunk 1 above)
Comment (joecrowleygaia) [reply abis_mapping/templates/survey_occurrence_data/templates/instructions.md]: fixed

File: abis_mapping/templates/survey_occurrence_data/templates/instructions.md (hunk 2 above)
Comment (joecrowleygaia) [reply abis_mapping/templates/survey_occurrence_data/templates/instructions.md]: fixed

File: abis_mapping/templates/survey_occurrence_data/templates/instructions.md (hunk 3 above)
Comment (joecrowleygaia) [reply abis_mapping/templates/survey_occurrence_data/templates/instructions.md]: fixed

**Comment by joecrowleygaia** [comment]
> I have also noticed that the schema example for the field `threatStatusCheckProtocol` there's a link that is not properly formatted. It contains spaces: `http://www.environmen t.gov.au/cgi-bin/sprat/p ublic/sprat.pl`
> 
> https://github.com/gaiaresources/abis-mapping/blob/bf36190055339a9fed76f5fc31a7d08189d1f2b6/abis_mapping/templates/survey_occurrence_data/schema.json#L611

Thanks serge, an issue caused through copying and pasting from PDF documents. I thought i had fixed all these but apparently a few were still lingering.

**Review by serge-gaia — APPROVED** [review]

**Pull request merged**